actors/common/src/main/java/org/sunbird/learner/actors/OrganisationManagementActor.java

```diff
@@ -287,9 +287,8 @@ private void createOrg(Request actorMessage) {
       Util.DbInfo orgDbInfo = Util.dbInfoMap.get(JsonKey.ORG_DB);
 
       validateChannelIdForRootOrg(request);
-      if (request.containsKey(JsonKey.CHANNEL)) {
-        validateChannel(request);
-      }
+      validateChannelIdForSubOrg(request);
```

Comment:
Combine validateChannelIdForRootOrg and validateChannelIdForSubOrg. 

If missing then simply throw mandatoryParamsMissing.